tf2/src/buffer_core.cpp

```diff
@@ -1259,7 +1259,6 @@ void BufferCore::removeTransformableCallback(TransformableCallbackHandle handle)
   {
     boost::mutex::scoped_lock lock(transformable_requests_mutex_);
     V_TransformableRequest::iterator it = std::remove_if(transformable_requests_.begin(), transformable_requests_.end(), RemoveRequestByCallback(handle));
-    transformable_requests_.erase(it, transformable_requests_.end());
```

Comment:
I don't see the need for this extra `erase()`: `std::remove_if` already removes all matches.
[According to the docs](https://en.cppreference.com/w/cpp/algorithm/remove#Return_value), it might even happen that the returned iterator `it` is invalid.